CONTRIBUTING.md

```diff
@@ -58,6 +58,23 @@ $ tox -e fuzz
 $ black-primer [-k -w /tmp/black_test_repos]
 ```
 
+### News / Changelog Requirement
+
+`Black` has CI that will check for an entry corresponding to your PR in `CHANGES.md`. If
+you feel this PR not require a `CHANGES.md` please state that in a comment and a
+maintainer can add a `skip_news` tag to make the CI pass. Otherwise, please ensure you
```

Comment:
last time I checked, it was `skip news` not `skip_news`, also what's up with the tag terminology?, I first think of Git tags before realizing my mistake, let's use labels instead if you don't mind :)

```suggestion
maintainer can add a `skip news` label to make the CI pass. Otherwise, please ensure you
```